package.json

```diff
@@ -62,10 +63,10 @@
     "@massalabs/as": "1.1.1-dev.20221215083749",
     "@massalabs/as-transformer": "^0.1.2",
     "@massalabs/as-types": "^0.0.3-dev",
-    "@massalabs/massa-as-sdk": "^2.5.0",
+    "@massalabs/massa-as-sdk": "file:///home/sydh/dev/massa-as-sdk",
```

Comment:
it's a local file ?

Reply:
I'm waiting for this PR to be merged: https://github.com/massalabs/massa-as-sdk/pull/313